### PR TITLE
Add test for installing the package

### LINKED ISSUE #27: Add a test for installing the package
Something like https://github.com/mozilla/libmozdata/blob/master/.travis.yml#L12.

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #27

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ install:
 script:
   - export PATH=$PWD/symlinks:$PATH
   - flake8 .
+  - python setup.py sdist bdist_wheel
   - python -m unittest discover tests
```
Comment (marco-c) [inline .travis.yml]: Can you add it at the end, after the tests?

Comment (marco-c) [inline .travis.yml]: Can you add it at the end, after the tests?

Comment (marco-c) [reply .travis.yml]: And we also need something like https://github.com/mozilla/libmozdata/blob/master/.travis.yml#L13, to make sure the package is installable via pip.

**Comment by rhcu** [comment]
That's weird but 2.7 build passed and 3.5 - not.

**Comment by marco-c** [comment]
There's a GitHub API rate limit which sometimes makes the test fail, the failure is unrelated to your PR.

**Pull request merged**